hazelcast/src/main/java/com/hazelcast/internal/dynamicconfig/ConfigurationService.java

```diff
@@ -326,4 +327,8 @@
      * @return registered MapEventJournal configurations
      */
     Map<String, EventJournalConfig> getMapEventJournalConfigs();
+
+    ReliableIdGeneratorConfig findReliableIdGeneratorConfig(String baseName);
```

Comment:
Move the `find...` method to the group of `find` methods above. Also some javadoc would be nice on the two new methods of this interface.